test/Parse/ConditionalCompilation/if_darwin.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %swift -target arm64-apple-ios8.0 -disable-legacy-type-info -parse-stdlib -enable-objc-interop -disable-objc-attr-requires-foundation-module -I %S/Inputs/usr/include -emit-ir %s -o - | %FileCheck %s -check-prefix CHECK-MACHO
```

Comment:
Are all of these compiler flags needed? `-parse-stdlib` in particular seems like it shouldn't be needed

Reply:
Should be fixed now